contrib/podex/podex.go

```diff
@@ -28,26 +28,26 @@ limitations under the License.
 package main
 
 import (
-	"bytes"
 	"encoding/json"
 	"flag"
 	"fmt"
-	"io/ioutil"
 	"log"
 	"net/http"
 	"os"
 	"strconv"
 	"strings"
 
-	"github.com/GoogleCloudPlatform/kubernetes/pkg/api/v1beta1"
 	"github.com/ghodss/yaml"
+	goyaml "gopkg.in/v2/yaml"
 )
 
-const usage = "usage: podex [-json|-yaml] [-id PODNAME] IMAGES"
+const usage = "usage: podex [-yaml|-json] [-pod|-container] [-id PODNAME] IMAGES..."
 
+var generateYAML = flag.Bool("yaml", true, "generate yaml manifest (default)")
```

Comment:
Ack, I find the `pod -json foo` rather than `pod -format=json foo` UI more friendly, but I agree it can also makes it confusing if we remove the explicit error checking the previous impl had.